apps/website/src/data/tech/presets.ts

```diff
@@ -1,393 +1,71 @@
-      belowplatforml: { description: "below the viewing platform" },
-      belowrampl: { description: "below the viewing platform to the right" },
-      belowramplz: { description: "below the den2 left ramp" },
-      belowrampr: { description: "zoomed in view below the den2 left ramp" },
-      brushl: { description: "below the den2 right ramp" },
-      center: { description: "center of the enclosure" },
-      den1: { description: "den1 near the pond" },
-      den2: { description: "den2 close to camera/viewing platform" },
-      den2b: { description: "bottom platform of den2" },
-      den2door: { description: "door to den2" },
-      den2entrace: { description: "front of den2" },
-      den2inside: { description: "view of inside den2" },
-      den2l: { description: "left of den2" },
-      den2m: { description: "middle platform of den2" },
-      den2shade: { description: "shade below the den2 platform" },
-      den2t: { description: "top platform of den2" },
-      down: { description: "straight down" },
-      downleft: { description: "down to the left" },
-      downright: { description: "down to the right" },
-      grass: { description: "middle grass patch" },
-      grassr: { description: "right side of middle grass patch" },
-      left: { description: "left side of enclosure" },
-      leftcorner: { description: "corner of the inside building" },
-      leftfence: { description: "view down the left fence" },
-      leftw: { description: "wide view of the left side" },
-      nearcorner: { description: "corner below the viewing platform" },
-      rampl: { description: "left ramp on den2" },
-      rampr: { description: "right ramp on den2" },
-      right: { description: "right of den2" },
-      rightcorner: { description: "right corner along the back fence" },
-      rightfence: { description: "view down the right fence" },
-      river: { description: "river near den1" },
-      water: { description: "drinking tub under the viewing platform" },
-      wolfcam: { description: "view of the main wolf camera" },
-    },
-  },
-  wolfindoor: {
-    title: "Wolf Indoor",
-    presets: {
-      doors: { description: "wolf doors" },
-      left: { description: "left side of the enclosure" },
-      right: { description: "right side of the enclosure" },
-      platformcenter: { description: "center platform" },
-      platformleft: { description: "left platform" },
-      platformright: { description: "right platform" },
-    },
-  },
-};
+const presets = {
+  pasture,
+  fox,
+  foxcorner,
+  roaches,
+  crowin,
+  crowout,
+  marmin,
+  marmout,
+  parrot,
+  wolf,
+  wolfcorner,
+  wolfindoor,
+  wolfswitch,
+  wolfden,
+  wolfden2,
+  hank,
+  pushpop,
+  pushpopindoor,
+  pushpopcrunch,
+  toast,
+  patchy,
+  noodle,
+  chicken,
+  chickenindoor,
+  georgie,
+  georgiewater,
+} as const satisfies Record<string, Camera>;
 
+export type Preset = keyof typeof presets;
+export type { PresetData, Camera };
```

Comment:
Can we make these types as exported inline where they're defined, if they're needed elsewhere?